src/components/Body.js

```diff
@@ -0,0 +1,9 @@
+import BaseComponent from "./BaseComponent.js";
+
+
+export default class Body extends BaseComponent {
+    constructor(page) {
+        super(page, page.locator('body'));
+        this.signUpButton =  page.locator('button', {hasText: 'Sign Up'})
+    }
+}
```

Comment:
You should associate your components with components on a page but not with HTML

Reply:
I do not think you need it